comment_parser/parsers/common.py

```diff
@@ -29,37 +29,37 @@ def __init__(self, text, line_number, multiline=False):
     self._line_number = line_number
     self._multiline = multiline
 
-  def text(self):
+  def text(self) -> str:
     """Returns the comment's text.
 
     Returns:
       String
     """
     return self._text
 
-  def line_number(self):
+  def line_number(self) -> int:
     """Returns the line number the comment was found on.
 
     Returns:
       Int
     """
     return self._line_number
 
-  def is_multiline(self):
+  def is_multiline(self) -> bool:
     """Returns whether this comment was a multiline comment.
 
     Returns:
       True if comment was a multiline comment, False if not.
     """
     return self._multiline
 
-  def __str__(self):
+  def __str__(self) -> str:
     return self._text
 
-  def __repr__(self):
+  def __repr__(self) -> str:
     return f'Comment({self._text}, {self._line_number}, {self._multiline})'
 
-  def __eq__(self, other):
+  def __eq__(self: 'Comment', other: object) -> bool:
```

Comment:
Generally we don't put type annotations on "self". Why put it here and not on the other instances of self however?

Reply:
:sweat_smile: seems like I was a bit over zealous - removed it